src/ignore-whitespace/ignore-whitespace.js

```diff
@@ -10,3 +9,10 @@ export default function init(prRow: Element) {
     url.search = searchParams.toString()
     link.href = url.href
 }
+
+export function ignoreWhitespaceInit() {
+    const toggle = document.querySelector('[data-key="ignoreWhitespace"]')
+    if (toggle && toggle.getAttribute('aria-checked') === 'false') {
+        toggle.click()
+    }
+}
```

Comment:
Is this element guaranteed to be in the page by the time the extension loads? I'm not sure if that's the case given that the diffs load asynchronously, so it might be worth covering that edge case here.

```suggestion
export async function ignoreWhitespaceInit() {
    const toggle: HTMLElement = await elementReady('[data-key="ignoreWhitespace"]')
    if (toggle && toggle.getAttribute('aria-checked') === 'false') {
        toggle.click()
    }
}
```

What do you think?

Reply:
updated and also move the function `main.js` from `pullrequestRelatedFeatures` to `codeReviewFeatures` to be accessible in commit page #311